### PR TITLE
Correcting the 'page-not-found' redirections

### PR DESCRIPTION
**Pull request by chamindi-a**

### CODE REVIEW

File: en/docs/develop/streaming-apps/working-with-the-design-view.md [hunk 1]
```diff
@@ -1312,9 +1312,9 @@ connections between Siddhi components. A connection is defined by
 drawing an arrow from one component to another by dragging the cursor as
 demonstrated below.
 
-![Connecting Siddhi components]({{base_path}}/assets/img/streaming/working-with-the design-view/connect-siddhi-components.gif)
+![Connecting Siddhi components](/assets/img/streaming/working-with-the design-view/Connect-Siddhi-Components.gif)              ]
```
Comment (npamudika) [inline en/docs/develop/streaming-apps/working-with-the-design-view.md:1315]: We don't need the closing square bracket at the end.

File: en/docs/integrate/develop/creating-artifacts/creating-a-message-processor.md [hunk 2]
```diff
@@ -64,4 +64,4 @@ See the links given below for descriptions of properties for each processor type
 
 ## Tutorials
 
--	See the tutorial on [using message stores and processors]({{base_path}}/integrate/tutorials/storing-and-forwarding-messages)
+-	See the tutorial on [using message stores and processors]({{base_path}}//tutorials/integration-tutorials/storing-and-forwarding-messages)
```
Comment (npamudika) [inline en/docs/integrate/develop/creating-artifacts/creating-a-message-processor.md:67]: Additional slash is there

File: en/docs/integrate/develop/creating-artifacts/creating-scheduled-task.md [hunk 3]
```diff
@@ -51,4 +51,4 @@ Update the task properties to specify the incoming message that should trigger t
 
 ## Tutorials
 
--   See the tutorial on [periodically executing an integration process]({{base_path}}/integrate/tutorials/using-scheduled-tasks) using a scheduled task
+-   See the tutorial on [periodically executing an integration process]({{base_path}}//tutorials/integration-tutorials/using-scheduled-tasks) using a scheduled task
```
Comment (npamudika) [inline en/docs/integrate/develop/creating-artifacts/creating-scheduled-task.md:54]: Please remove the additional slash before 'tutorials'

Comment (chamindi-a) [reply en/docs/integrate/develop/creating-artifacts/creating-scheduled-task.md]: fixed with [33b1850](https://github.com/wso2/docs-apim/pull/7321/commits/33b1850d15ef4f967024552b40b35169ff6bc33c)

File: en/docs/integrate/develop/creating-artifacts/creating-a-message-processor.md (hunk 2 above)
Comment (chamindi-a) [reply en/docs/integrate/develop/creating-artifacts/creating-a-message-processor.md]: fixed with [33b1850](https://github.com/wso2/docs-apim/pull/7321/commits/33b1850d15ef4f967024552b40b35169ff6bc33c)

File: en/docs/develop/streaming-apps/working-with-the-design-view.md [hunk 4]
```diff
@@ -1312,9 +1312,9 @@ connections between Siddhi components. A connection is defined by
...
 
-## Saving and running Siddhi applications
+    ## Saving and running Siddhi applications
```
[the remark this answers is not in the document]
Comment (chamindi-a) [reply en/docs/develop/streaming-apps/working-with-the-design-view.md]: fixed with [33b1850](https://github.com/wso2/docs-apim/pull/7321/commits/33b1850d15ef4f967024552b40b35169ff6bc33c)

File: en/docs/develop/streaming-apps/working-with-the-design-view.md (hunk 1 above)
Comment (chamindi-a) [reply en/docs/develop/streaming-apps/working-with-the-design-view.md]: fixed with [33b1850](https://github.com/wso2/docs-apim/pull/7321/commits/33b1850d15ef4f967024552b40b35169ff6bc33c)

File: en/docs/develop/streaming-apps/working-with-the-design-view.md [hunk 5]
```diff
@@ -1310,11 +1310,11 @@ In order to define how the Siddhi components in a Siddhi application
 interact with each other to process events, you need to define
 connections between Siddhi components. A connection is defined by
 drawing an arrow from one component to another by dragging the cursor as
-demonstrated below.
+running Siddhi applications
```
Comment (npamudika) [inline en/docs/develop/streaming-apps/working-with-the-design-view.md:1313]: Is this change required?

Comment (chamindi-a) [reply en/docs/develop/streaming-apps/working-with-the-design-view.md]: Changed [0f817e3](https://github.com/wso2/docs-apim/pull/7321/commits/0f817e392b5868be6f6529c83d501fb8e2ac9c48)

**Review by npamudika — APPROVED** [review]
